src/components/services/FreshRSS.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <Generic :item="item">
+      <template #indicator>
+        <div class="notifs">
+          <strong v-if="subscriptions > 0" class="notif subscriptions" title="Subscriptions">
+            {{ subscriptions }}
+          </strong>
+          <strong v-if="unread > 0" class="notif unread" title="Unread">
+            {{ unread }}
+          </strong>
+          <strong
+            v-if="serverError"
+            class="notif errors"
+            title="Connection error to the FreshRSS API, check url username and password in config.yml"
+            >?</strong
+          >
+        </div>
+      </template>
+    </Generic>
+  </template>
+  
+  <script>
+  import service from "@/mixins/service.js";
+  import Generic from "./Generic.vue";
+  
+  export default {
+    name: "FreshRSS",
+    mixins: [service],
+    props: {
+      item: Object,
+    },
+    components: {
+      Generic,
+    },
+    data: () => {
+      return {
+        subscriptions: null,
+        unread: null,
+        serverError: false,
+      };
+    },
+    created: function () {
+      const updateInterval = parseInt(this.item.updateInterval, 10) || 0;
+      if (updateInterval > 0)
+        setInterval(() => this.fetchConfig(), updateInterval);
+      
+      this.fetchConfig();
+    },
+    methods: {
+      fetchConfig: async function () {
+  
+        if (!this.auth) {
+          const match = await this.fetch(`/api/greader.php/accounts/ClientLogin?Email=${this.item.username}&Passwd=${this.item.password}` , { method: 'GET', cache: "no-cache" }, false )
+           .then(response => {return response.text();})
+           .then(body => {return body.match(/Auth=(([([a-z0-9]+)\/([([a-z0-9]+))/i);});
+          if (match !== null)
+            this.auth = match[1];
+        }                     
+  
+        const headers = {
+          "Authorization": `GoogleLogin auth=${this.auth}`,
+        }; 
+  
+        this.fetch(`/api/greader.php/reader/api/0/subscription/list?output=json`, { headers })
+          .then((subscription) => {
+            this.subscriptions = 0;
```

Comment:
You should be able to set `subscriptions` to 0 in the declaration, line 37, and remove this line